invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -96,31 +97,7 @@ def last_updated(self, obj):
 
     def get_locations(self, obj):
         """Get locations."""
-        locations = []
-
-        access_location = obj["metadata"].get("locations", [])
-        if not access_location:
-            return missing
-
-        for location in access_location.get("features", []):
-            location_string = ""
-
-            place = location.get("place")
-            description = location.get("description")
-            if place:
-                location_string += f"name={place}; "
-            if description:
-                location_string += f"description={description}"
-            geometry = location.get("geometry")
-            if geometry:
-                geo_type = geometry["type"]
-                if geo_type == "Point":
-                    coords = geometry["coordinates"]
-                    location_string += f"; lat={coords[0]}; lon={coords[1]}"
-
-            locations.append(location_string)
-
-        return locations
+        return DublinCoreSchema.get_locations(obj)
```

Comment:
major: I see the fact that the logic inside is the same (very nice catch!), however I would try avoid making schemas dependent of each other. If needed, there could be a `schemas.py` or `fields.py` in the `serializers/` root folder (i.e. one level up than currently). 

cc @zzacharo

Reply:
sure that seems nicer
I'll get to work on it thx :)

Reply:
Something like this?
Note: some functions found in `schemas.py` are only used once
This is because the BibTex serializer which I'm working on would also call these functions (hence 2 calls)